docs_guidelines/tutorials/new_doc_project.rst

```diff
@@ -0,0 +1,610 @@
+    translate text that it generates into that language.
+
+    For a list of supported codes, see
+    http://sphinx-doc.org/config.html#confval-language.
+    > Project language [en]:
+
+Here the language will be English, that is, the default.
+
+.. code-block:: text
+
+    Creating file ./docs/conf.py.
+    Creating file ./docs/index.rst.
+    Creating file ./docs/Makefile.
+    Creating file ./docs/make.bat.
+
+    Finished: An initial directory structure has been created.
+
+The new folder will consist of:
+
+* ``index.rst``: the RST file that will make up the home page when built as HTML.
+* ``conf.py``: a file that includes all the Sphinx configuration settings.
+* ``Makefile`` (or ``make.bat`` for Windows): files that enable you to build documentation using ``make``.
+* ``_templates``: a folder for your own HTML templates (now empty).
+* ``_static``: a folder for static files like images (now empty).
+* ``_build``: a folder for built documentation (now empty).
+
+Even though you have created a working Sphinx documentation folder, you are not using the ``qiskit_sphinx_theme`` yet. In order to do that, you have to open ``conf.py``
+and change the value of the variable ``html_theme`` from ``'alabaster'`` to ``'qiskit_sphinx_theme'``.
+
+Structure your documentation
+============================
+
+In Qiskit we are following the `Diataxis <https://diataxis.fr/>`_ documentation framework, that means that our documentation has to be separated into:
+
+* `Tutorials <https://diataxis.fr/tutorials/>`_.
+* `How-to guides <https://diataxis.fr/how-to-guides/>`_.
+* `API reference <https://diataxis.fr/reference/>`_.
+* `Explanations <https://diataxis.fr/explanation/>`_.
+
+In order to do that, you have to create 4 new folders inside ``docs``, that you will call ``tutorials``, ``how_to``, ``apidocs`` and ``explanations``. Inside each one of them, a file called ``index.rst`` must be created.
+
+Apart from that, you should add your release notes to ``docs`` as a file called ``release_notes.rst`` and a getting started guide called ``getting_started.rst``.
+
+Sidebar
+=======
+
+Now that you have all the needed ``.rst`` files, you can create a sidebar for your documentation page. This can be done by linking to the files with a `toctree <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-toctree>`_
+in ``docs/index.rst``.  Your ``index.rst`` should look like this:
+
+.. code-block:: text
+
+    =====================================
+    Welcome to Qiskit X's documentation!
+    =====================================
+
+    Overview
+    ========
+
+    Explain your package here.
+
+
+    .. toctree::
+    :hidden:
+
+    Overview <self>
+    Getting Started <getting_started>
+    Tutorials <tutorials/index>
+    How-to Guides <how_to/index>
+    API Reference <apidocs/index>
+    Explanations <explanations/index>
+    Release Notes <release_notes>
+    GitHub <https://github.com/Qiskit/qiskit_x>
+
```

Comment:
I have a how-to about just that in #98. It might be better to link to it (if it gets merged of course) to avoid duplication of content.

I agree with mentioning the idea of building locally to check the output but should that be a proper step? For me it fits a bit better as a note/warning.